config/default.json

```diff
@@ -73,6 +73,11 @@
             "location": "autoAcceptPendingRelationships/AutoAcceptPendingRelationshipsModule",
             "enabled": false
         },
+        "autoDecomposeDeletionProposedRelationships": {
```

Comment:
What do y'all think about shortening the name of this feature to "AutoDecomposeRelationships"? I feel like the rest is redundant.

Reply:
verbosity != redundancy.

Reply:
Also i just stay in the naming scheme of the Auto..Modules

Reply:
And AutoDecomposeRelationships is not precise enough: it's also possible to decompose terminated relationships, so I would exept those Relationships also being decomposed.